database/seed.js

```diff
@@ -0,0 +1,52 @@
+
+/* eslint-disable */
+const mongoose = require('mongoose');
+const db = require('./index.js');
+const faker = require('faker');
+const image = require('./image.js');
+
+
+//create document for each image
+var imageDataCreater = function () {
+
+  var resultArr = [];
+
+  for(var i = 1; i < 101; i++){
+    resultArr.push({
+        title: 'Relax in the comfort of '+ faker.company.bsBuzz,
```

Comment:
@ykent4 What is `bsBuzz`? I did a quick google through Faker docs and the only example I found was "maximize."

That would leave the title as "Relax in the comfort of maximize," which I'm not sure indicates that bsBuzz is a great fit. Maybe I just found a bad example, though. Lemme know!

Reply:
return faker.random.arrayElement(faker.definitions.company.bs_verb);

The above return statement is from bsBuzz in the faker library. It seems to return a verb. Thanks for pointing out how that would be a very awkward string. I'm changing this to bsNoun.